src/crt/idvrmu_sdvrmu.src

```diff
@@ -1,5 +1,29 @@
 	assume	adl=1
 
+	section	.text
+	public	__sdvrmu
+__sdvrmu:
+; I: HL=dividend, BC=divisor
+; O: a=0, ude=HL/BC, uhl=HL%BC
+
+	push	hl
+	dec	sp
+	pop	de
+	inc	sp
+	ld	e, 0
+
+	inc	bc
+	dec.s	bc
+
+	push	af
+	ld	a, 16
+
+if PREFER_OS_CRT
+	require	__idvrmu_common
```

Comment:
It looks like this was done improperly by me to begin with, but if `PREFER_OS_CRT` is set, I think we should probably just zero-extend `HL` and `BC` and then jump to `__idvrmu`.